lte/gateway/fabfile.py

```diff
@@ -159,21 +159,29 @@ def package(
 @task
 def open_orc8r_port_in_vagrant(c):
     """
-    Add a line to Vagrantfile file to open 9445 port on Vagrant.
-    Note that localhost request to 9443 will be sent to Vagrant vm.
-    Remove this line manually if you intend to run orc8r on your host
+    Add lines to the Vagrantfile file to open 9445 port for relevant VMs.
+    Note that localhost request to 9443 will be sent to a Vagrant vm.
+    Remove this line manually if you intend to run orc8r on your host.
     """
-    cmd_yes_if_exists = """grep -q 'guest: 9443, host: 9443' Vagrantfile"""
+    is_port_open = "grep -q 'guest: 9443, host: 9443' Vagrantfile"
 
-    # Insert line after a specific line
-    cmd_insert_line = \
-        "awk '/config.vm.define :magma,/{print;print " \
-        "\"    magma.vm.network \\\"forwarded_port\\\", " \
-        "guest: 9443, host: 9443\"" \
-        ";next}1' Vagrantfile >> Vagrantfile.bak00 && " \
-        "cp Vagrantfile.bak00 Vagrantfile && rm Vagrantfile.bak00"
+    pattern_template = "^  config.vm.define :{},.*$"
+    open_port_template = '    {}.vm.network \"forwarded_port\", guest: 9443, host: 9443'
+    # workaround:
+    # 1. duplicate line ("p") and then override - because "a" (append text
+    #    after line) works different on linux and macos (but would be more elegant)
+    # 2. don't use -i (inline replace), but copy .bak file - because syntax
+    #    differs between linux and macos
+    cmd_open_port_template = "sed '/{}/p; s/{}/{}/' Vagrantfile > v.bak && cp v.bak Vagrantfile && rm v.bak"
 
-    c.run(f"{cmd_yes_if_exists} || ({cmd_insert_line})")
+    def create_cmd(machine):
+        pattern = pattern_template.format(machine)
+        open_port_line = open_port_template.format(machine)
+        return cmd_open_port_template.format(pattern, pattern, open_port_line)
+
+    cmd_open_port = f"{create_cmd('magma')} && {create_cmd('magma_deb')}"
```

Comment:
Why do we need this for both machines? This is only called from the FEG integ tests which now run on `magma_deb`.

Reply:
... first I wanted to keep the option open to use magma or magma_deb ... and here I kept it for that reason. But, yeah, you are right ... I will remove it.

Reply:
But I suggest to keep the templating logic - imho it's better readable then hardcoding "magma_deb" in a long string.